README.md

```diff
@@ -1,48 +1,95 @@
-This dashboard can be transferred to Grafana using the [Grafana Agent](https://charmhub.io/grafana-agent), which consumes the `cos-agent` integration.
+with a cloud configuration (for the OpenStack tenant used to spawn runner VM's) in `clouds.yaml`:
+
+```yaml
+clouds:
+  <cloud-name>:
+    auth:
+      auth_url: <keystone-auth-url>
+      project_name: <project>
+      username: <username>
+      password: <password>
+      user_domain_name: <user-domain-name>
+      project_domain_name: <project-domain-name>
+    region_name: <region>
+  ```
+
+Assuming you have already deployed the [Github Runner Image Builder](https://charmhub.io/github-runner-image-builder) charm
+with the name `github-runner-image-builder`, you can use the following command to integrate it with the GitHub Runner charm:
+
+```bash
+juju integrate github-runner-image-builder github-runner
+```
 
-## Development
+You can scale the amount of virtual machines using
 
-This charm uses black and flake8 for formatting. Both run with the lint stage of tox.
+```bash
+juju config github-runner virtual-machines=5
+```
 
-## Testing
+You can change the reconciliation interval, to e.g. 5 minutes, using
 
-Testing is run via tox and pytest. The unit test can be ran with `tox -e unit` and the integration test on juju 3.1 with `tox -e integration-juju3.1`.
+```bash
+juju config github-runner reconciliation-interval=5
+```
 
-Dependencies are installed in virtual environments. Integration testing requires a juju controller to execute. These tests will use the existing controller, creating an ephemeral model for the tests which is removed after testing. If you do not already have a controller setup, you can configure a local instance via LXD, see the [upstream documentation](https://juju.is/docs/lxd-cloud) for details.
+You can trigger reconciliation manually using an action (assuming the unit is named `github-runner/0`):
 
-## Generating src docs for every commit
+```bash
+juju run github-runner/0 reconcile-runners
+```
 
-Run the following command:
+If you need to flush and replace the runners with a new set of runners, you can use the following command:
 
 ```bash
-echo -e "tox -e src-docs\ngit add src-docs\n" >> .git/hooks/pre-commit
-chmod +x .git/hooks/pre-commit
+juju run github-runner/0 flush-runners
 ```
+
+
+## Integrations
+The charm supports [multiple integrations](https://charmhub.io/github-runner/integrations),
+but in order to deploy the charm using OpenStack VM's for the runners, you need it to relate it
+with an image-builder using the [image](https://charmhub.io/github-runner/integrations#image) 
+endpoint. Via this integration, the charm detects the supported images to use for spawning the virtual machines.
+
+
+## Learn more
+* [Read more](https://charmhub.io/github-runner)
+* [Developer documentation](https://charmhub.io/github-runner/docs/how-to-contribute)
+
+## Project and community
+* [Issues](https://github.com/canonical/github-runner-operator/issues)
```

Comment:
I think it would be good to have a "Contributing" point here that repeats the link you used for developer documentation.